x-pack/plugins/security_solution/public/common/components/ml/anomaly/use_anomalies_search.test.ts

```diff
@@ -132,40 +132,6 @@ describe('useNotableAnomaliesSearch', () => {
     });
   });
 
-  it('returns uninstalled jobs', async () => {
```

Comment:
This use case doesn't make sense anymore because uninstalled jobs are also returned from the API.